image-fetcher/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: build push deploy
+
+images: # ../notebook/notebook-worker-image
+	cat /dev/null $< > images
```

Comment:
Hmm.  We should think about these cross-project dependencies.  I'm not sure I have anything smart to say here.  If one gets updated without the other, we might have a bad time.

Another alternative is to the puller to get the image from the deployments pod template images.

Reply:
Yeah. I'm beginning to think we should have one top-level Makefile so we can properly describe cross-project dependencies.

Reply:
This is currently an issue with testing CI locally, it's very easy to forget to update your local install of `batch`.